TECHNOLOGY_CHOICES.md

```diff
@@ -0,0 +1,104 @@
+# Technology Choices
+
+# Databases
+- PostgreSQL
+- Neo4J
+- CouchDB
+- eXist-db
+
+## Checks for determining database
+- Database should be able to store XML data
+- Database should be relational, because we need relations between files/pieces of files/models
+- Database must be provided with the solutions, for example part of a Docker image
+- Support must be available, in case of trouble.
+- Database must be open-source, because the whole solution must be open-source
+
```

Comment:
An important requirement is how well the database integrates with code, since that is what you intend to do. When you choose the known ones like Postgres, it easily integrates with Java code for (almost) free with patterns like dao (https://www.baeldung.com/java-dao-pattern). It is good to evaluate your db of choice with respect to this.

Reply:
Absolutely, BaseX for example integrates very well with all kinds of languages:
https://docs.basex.org/wiki/Clients